app/src/main/java/org/kiwix/kiwixmobile/utils/ImageUtils.java

```diff
@@ -1,39 +1,49 @@
 package org.kiwix.kiwixmobile.utils;
 
 import android.graphics.Bitmap;
-import android.util.Log;
+import android.media.ThumbnailUtils;
 import android.view.View;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 
 public class ImageUtils {
```

Comment:
this class is a perfect chance to convert to kotlin, one function bunch of hideous java null checking malarkey

Reply:
Hmmm. I'll try to do that. But I just know a bit of Kotlin.

Reply:
I've been doing it for years, here to offer help if you need it. `ctrl+shift+alt+k` to convert to kotlin if you didn't already know

Reply:
@abdulwd do you have availability to do this next week?

Reply:
I'm not sure. You can pick it up if you're free.

Reply:
May more painful to rewrite than it had any right to be. Give it a review I guess?

Reply:
@abdulwd I'll probably just merge this tomorrow if you don't have any feedback